__tests__/app.test.js

```diff
@@ -355,22 +399,50 @@ describe('PATCH /api/articles/:article_id', () => {
       test('PATCH: 400 - returns an error when new votes is not a number', () => {
 
         const newVotes = { 
-          new_votes: 'author'
+          inc_votes: 'author'
         }
 
         return request(app)
             .patch('/api/articles/2')
             .send(newVotes)
             .expect(400)
             .then(({ body }) => {
-                expect(body.msg).toBe('Bad Request: votes must be a number');
+                expect(body.msg).toBe('Invalid Input');
             });
     });
 
+    test('PATCH: 400 - returns an error when passed nothing into inc votes', () => {
+
+      const newVotes = {}
+
+      return request(app)
+          .patch('/api/articles/2')
+          .send(newVotes)
+          .expect(400)
+          .then(({ body }) => {
+              expect(body.msg).toBe('Invalid Input');
+          });
+  });
```

Comment:
This is a 23502 error, better to send a different error message like "required key missing" - It'll help with your frontend project debugging